com.ibm.wala.cast.java/src/com/ibm/wala/cast/java/examples/ast/SynchronizedBlockDuplicator.java

```diff
@@ -76,7 +77,7 @@ public boolean equals(Object o) {
       return (o instanceof UnwindKey)
           && ((UnwindKey) o).testDirection == testDirection
           && ((UnwindKey) o).syncNode == syncNode
-          && (rest == null ? ((UnwindKey) o).rest == null : rest.equals(((UnwindKey) o).rest));
+          && Objects.equals(rest, ((UnwindKey) o).rest);
```

Comment:
👍 I didn't know about this!

Reply:
Yeah, `java.util.Objects.equals` is a nice little helper for a very common code pattern.